improver_tests/utilities/cube_manipulation/test_maximum_in_height.py

```diff
@@ -65,18 +66,24 @@ def wet_bulb_temperature() -> Cube:
         (50, 1000, [300, 400, 300]),
     ),
 )
-def test_maximum_in_height(lower_bound, upper_bound, expected, wet_bulb_temperature):
+def test_maximum_in_height(
+    lower_bound, upper_bound, expected, wet_bulb_temperature, new_name
+):
     """Test that the maximum over the height coordinate is correctly calculated for
-    different combinations of upper and lower bounds."""
+    different combinations of upper and lower bounds. Also checks the name of the
+    cube is correctly updated."""
 
-    result = maximum_in_height(
-        wet_bulb_temperature,
-        lower_height_bound=lower_bound,
-        upper_height_bound=upper_bound,
-    )
+    if new_name:
+        expected_name = new_name
+        result = maximum_in_height(
+            wet_bulb_temperature, lower_bound, upper_bound, new_name
+        )
+    else:
+        expected_name = "wet_bulb_temperature"
+        result = maximum_in_height(wet_bulb_temperature, lower_bound, upper_bound)
```

Comment:
I think this can be reduced in length a bit:

```suggestion
    expected_name = "wet_bulb_temperature"
    if new_name:
        expected_name = new_name
    result = maximum_in_height(
        wet_bulb_temperature, lower_bound, upper_bound, new_name
    )
```

because if `new_name = None` the plugin doesn't change the name.

Reply:
Quite right. I've tidied it up as suggested.